WebServices/AzureSignalR/ChatClient/MainPage.xaml.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace ChatClient
+{
+    public partial class MainPage : ContentPage
```

Comment:
Normally we don't use the `private` keyword, as fields/methods are private by default if an accessibility isn't specified. However, I suspect this is something that's come over from the Mono coding standards, and isn't an official DevDiv position. But it's relatively rare to see it in a Xamarin sample.

I'm not saying you should remove all the `private` keywords. Just an issue to be aware of if it comes up in future.

Reply:
Strange. I don't usually use the `private` keyword but I randomly did here. I wonder if there was a method already decorated with `private` and I just copied the style unconsciously. I removed them on principal.